package.json

```diff
@@ -24,7 +24,7 @@
     "test": "c8 mocha --recursive build/test/unit",
     "clean": "gts clean",
     "compile": "tsc -p .",
-    "postcompile": "cpy 'utils/**/*.*' build --parents && cpy 'test/**/*.*' build --parents",
```

Comment:
I can't quite imagine what this was trying to do

Reply:
The only thing I can think of is that perhaps there were non-ts files in utils that needed to be in the build directory that tsc wasn't processing.

Anyway, if it is not needed, we should delete it.  Thanks for catching this.